mbed_tools_ci_scripts/spdx_report/spdx_document.py

```diff
@@ -177,6 +177,11 @@ def external_refs(self) -> List[DependencySpdxDocumentRef]:
         """
         return self._other_document_references
 
+    @external_refs.setter
```

Comment:
Do we really need this to be a property? There's no encapsulation here, as we just pass `external_refs` directly to `_other_document_references`. Would it be better to just have `external_refs` as a public attribute and remove `_other_document_references`?

Reply:
True. I thought nevertheless that it made the documentation clearer with proper docstring for the field.
![image](https://user-images.githubusercontent.com/29707928/79448551-85f0c100-7fd9-11ea-8c33-4d153409242a.png)